tests/integ/integ_test.go

```diff
@@ -12,7 +12,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package integ
```

Comment:
I'd be +1 to use `integration`, since it's not too long.

Reply:
sure.